apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberClient.cs

```diff
@@ -421,6 +429,9 @@ internal SubscriberClientImpl(SubscriptionName subscriptionName, IEnumerable<Sub
         private CancellationTokenSource _globalSoftStopCts; // soft-stop is guarenteed to occur before hard-stop.
         private CancellationTokenSource _globalHardStopCts;
 
+        // For testing
```

Comment:
Could you describe what this means in the comment? It took me a while to "get" what was going on here (before this PR), and it would be nice for the next person not to have to take that time :)

Reply:
Done